jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -40,5 +40,33 @@
       @click:action="spectral_extraction"
     ></plugin-add-results>
 
+    <j-plugin-section-header v-if="export_enabled">Results</j-plugin-section-header>
```

Comment:
```suggestion
    <j-plugin-section-header v-if="extracted_spec_available && export_enabled">Results</j-plugin-section-header>
```

Reply:
same here, i kept it like this to be consistent with the moment map plugin. if you think this is better ill change it there too